persistence/trees/trees_test.go

```diff
@@ -23,28 +18,3 @@ func TestTreeStore_DebugClearAll(t *testing.T) {
 	// TODO: Write test case for the DebugClearAll method
 	t.Skip("TODO: Write test case for DebugClearAll method")
 }
-
-func TestStateTree_Operations(t *testing.T) {
```

Comment:
Are we bringing this back?

Reply:
Not exporting any more those functions dont exist rethinking how we should export trees atm in a way that doesnt alter them